package.json

```diff
@@ -39,6 +39,7 @@
     "null-loader": "^0.1.0",
     "postcss": "^4.0.2",
     "postcss-custom-properties": "3.0.1",
+    "prop-types": "^15.5.8",
```

Comment:
Should this & create-react-class be dependencies?  React-redux has them as such
https://github.com/reactjs/react-redux/blob/master/package.json